.github/workflows/core.yml

```diff
@@ -1,11 +1,14 @@
 name: core
+
 on:
   push:
+    branches:
```

Comment:
The GitHub CI pipeline is turned off with this configuration in developer branches within forks.

Reply:
I also thought so. Let me finding a solution so that it should be adopted in apache repository :-)

Reply:
I changed it to use `branches-ignores` instead. :-)